tests/test_metadata.py

```diff
@@ -1154,3 +1156,46 @@ def test_success(self):
         equivalent_identifiers = [x.output for x in identifier.equivalencies]
         eq_([book.primary_identifier], equivalent_identifiers)
     
+
+class TestMARCExtractor(DatabaseTest):
+
+    def setup(self):
+        super(TestMARCExtractor, self).setup()
+        base_path = os.path.split(__file__)[0]
+        self.resource_path = os.path.join(base_path, "files", "marc")
+
+    def sample_data(self, filename):
+        return open(os.path.join(self.resource_path, filename)).read()
```

Comment:
Can you return the file contents from within an `with open(...) as` block or use the `sample_data` method from `tests/__init__.py`, so that the file is closed after the fact?